eslint.config.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+ module.exports = {
+	env: {
+        browser: true
+	},
+	extends: [require.resolve('eslint-config-liferay'), require.resolve('eslint-config-prettier')],
+	plugins:['babel'],
+	globals: {
+        after: true,
+        assert: true,
+        before: true,
+        sinon: true
```

Comment:
This doesn't belong here. Your use of `env` in the `test` directory should be enough shouldn't it? And if you can't make that work, move this into a file in the `test` directory.

Reply:
When removing these ones some errors appears on log of eslint: 

```log
  ...
  
  1990:25  error  'sinon' is not defined   no-undef
  1998:7   error  'assert' is not defined  no-undef

  ...
```

These objects are being applied globally on Karma(or maybe gulp-metal). I think that we can remove it, when removing gulp-metal.

Reply:
Btw, I found that I can move these globals to `test` pattern override and I don't need more `after` and `before` declared as a global.